lms/views/exceptions.py

```diff
@@ -53,9 +53,9 @@ def validation_error(self):
     @exception_view_config(
         ReusedConsumerKey, renderer="lms:templates/error_dialog.html.jinja2"
```

Comment:
Note that the exception associated with this view is already called `ReusedConsumerKey` but the view was called `reused_tool_guid_error()`